libraries/postgres/test/database_test.js

```diff
@@ -101,6 +101,45 @@ dbSuite(path.basename(__filename), function() {
     assert.deepEqual(res.map(r => r.total).sort(), [16, 20]);
   });
 
+  test('procedure method names are unique', async function () {
+    const schema = Schema.fromSerializable({
+      versions: [{
+        version: 1,
+        migrationScript: `begin
+          create table testing (a integer, b integer);
+        end`,
+        methods: {
+          testdata: {
+            mode: 'write',
+            serviceName: 'service-1',
+            args: '',
+            returns: 'void',
+            body: `begin
+              insert into testing values (1, 2), (3, 4);
+            end`,
+          },
+          testdata: {
+            mode: 'read',
+            serviceName: 'service-2',
+            args: '',
+            returns: 'void',
+            body: `begin
+              insert into testing values (1, 2), (3, 4);
+            end`,
+          },
```

Comment:
I'm curious how this test case works: doesn't that create a JS object with just one property?

Reply:
oo, you're right. I actually haven't tested this. I was relying on CI to run the tests for me. It's probably a good time for me to read the README and set up the database.